AllASPFuncs.py

```diff
@@ -82,14 +83,15 @@ def local_grid(ra_center, dec_center, wcs, npoints, size = 25, spacing = 1.0, im
     difference = int((size - subsize)/2)
 
     x_center, y_center = wcs.toImage(ra_center, dec_center, units='deg')
+    x = np.arange(difference, subsize+difference, spacing)
```

Comment:
You could write this as
`x = difference + np.arange(0, subsize, spacing)`
which is arguably a little more compact and helps communicate more clearly what's being set up.